src/components/forms/Number.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormControl } from 'patternfly-react';
+import './Dropdown.css';
+import { eventValueHandler } from './utils';
+
+const onKeyDown = e => {
+  if (!((e.keyCode > 95 && e.keyCode < 106) || (e.keyCode > 47 && e.keyCode < 58) || e.keyCode === 8)) {
+    e.preventDefault();
+    return false;
+  }
+  return true;
+};
+
+export const Number = ({ id, value, defaultValue, onChange, onBlur, positive }) => (
+  <FormControl
+    id={id}
+    type="number"
```

Comment:
I assume this mostly works on newer browsers? (not that I mind) but just wondering what is the minimum browser we support?

Reply:
also, maybe this makes sense to have this component in pf-react.

Reply:
We should be alligned with tectonic console. This is what their github page says:

```
Supported Browsers
We support the latest versions of the following browsers:

Edge
Chrome
Safari
Firefox
IE 11 and earlier is not supported.
```

So I guess we are supporting just the latest browsers.

Reply:
I am working on the pf-react component